### PR TITLE
Add Vue2 Vuetify Renderer Set

### LINKED ISSUE #209: Document vue2-vuetify renderer set
* Add to overview graphic in landing page
 * Add to architecture docs image and text
 * Add to renderer sets
 * Add to vue integration page

It's a "new" renderer set for Vue 2 called `@jsonforms/vue2-vuetify`. It's not part of the core repository but lives here: https://github.com/eclipsesource/jsonforms-vuetify-renderers. It supports all the same features as React Material.

### PR DESCRIPTION
**Pull request by TheZoker**
Add Vue2 Vuetify Renderer Set

Closes #209

### CODE REVIEW

File: content/docs/renderer-sets.mdx [hunk 1]
```diff
@@ -32,3 +32,11 @@ The Angular Material UI Renderer Set is based on [Angular Material](https://mate
 
 The Vue Renderer Set is based on plain HTML with a set of custom styles.
 For more information, see the [Vue Vanilla Readme](https://github.com/eclipsesource/jsonforms/blob/master/packages/vue/vue-vanilla/README.md).
+
+## Vue2 Vuetify Renderer Set
+
+The Vue2 Vuetify Set provides a Vuetify based renderer set for JSON Forms Vue 2.
+It's not part of the core, but lives in it's own repository:
```
Comment (sdirix) [inline content/docs/renderer-sets.mdx]:
```suggestion
It's not part of the main repository, but lives in its own repository:
```

**Review by sdirix — CHANGES_REQUESTED** [review]
Vue2-Vuetify is missing a custom renderer box ;)

**Review by sdirix — CHANGES_REQUESTED** [review]
A mention of vue2 vuetify is missing in the architecture doc

**Comment by lucas-koehler** [comment]
@sdirix Can we merge this to conclude the website updates for the 3.0 release?

**Review by sdirix — APPROVED** [review]

**Pull request merged**